setting both kts to volt source mode
changing volt range on bias
bias keithley volt zero and output on
curr keithley volt zero and output on
testing device a1
seems like it's an open switch
end backoff 60.5, 1.45812e-11
[Base        ] Leaving application in progress...
08/01/2017 16:24:58.276, stderr: 
08/01/2017 16:24:58.280, stderr: Traceback (most recent call last):
08/01/2017 16:24:58.286, stderr:   File "C:\Users\MOLNAR_PH330_D1\Desktop\NZERO\python_code\pstest.py", line 641, in <module>
08/01/2017 16:24:58.290, stderr:     Nzero_Testing_AppApp().run()
08/01/2017 16:24:58.296, stderr:   File "C:\Python27\lib\site-packages\kivy\app.py", line 828, in run
08/01/2017 16:24:58.301, stderr:     runTouchApp()
08/01/2017 16:24:58.305, stderr:   File "C:\Python27\lib\site-packages\kivy\base.py", line 504, in runTouchApp
08/01/2017 16:24:58.309, stderr:     EventLoop.window.mainloop()
08/01/2017 16:24:58.312, stderr:   File "C:\Python27\lib\site-packages\kivy\core\window\window_sdl2.py", line 663, in mainloop
08/01/2017 16:24:58.315, stderr:     self._mainloop()
08/01/2017 16:24:58.321, stderr:   File "C:\Python27\lib\site-packages\kivy\core\window\window_sdl2.py", line 405, in _mainloop
08/01/2017 16:24:58.325, stderr:     EventLoop.idle()
08/01/2017 16:24:58.326, stderr:   File "C:\Python27\lib\site-packages\kivy\base.py", line 342, in idle
08/01/2017 16:24:58.328, stderr:     self.dispatch_input()
08/01/2017 16:24:58.329, stderr:   File "C:\Python27\lib\site-packages\kivy\base.py", line 327, in dispatch_input
08/01/2017 16:24:58.332, stderr:     post_dispatch_input(*pop(0))
08/01/2017 16:24:58.334, stderr:   File "C:\Python27\lib\site-packages\kivy\base.py", line 233, in post_dispatch_input
08/01/2017 16:24:58.335, stderr:     listener.dispatch('on_motion', etype, me)
08/01/2017 16:24:58.338, stderr:   File "kivy\_event.pyx", line 718, in kivy._event.EventDispatcher.dispatch (kivy\_event.c:8191)
08/01/2017 16:24:58.339, stderr:   File "C:\Python27\lib\site-packages\kivy\core\window\__init__.py", line 1188, in on_motion
08/01/2017 16:24:58.342, stderr:     self.dispatch('on_touch_down', me)
08/01/2017 16:24:58.344, stderr:   File "kivy\_event.pyx", line 718, in kivy._event.EventDispatcher.dispatch (kivy\_event.c:8191)
08/01/2017 16:24:58.345, stderr:   File "C:\Python27\lib\site-packages\kivy\core\window\__init__.py", line 1204, in on_touch_down
08/01/2017 16:24:58.348, stderr:     if w.dispatch('on_touch_down', touch):
08/01/2017 16:24:58.349, stderr:   File "kivy\_event.pyx", line 718, in kivy._event.EventDispatcher.dispatch (kivy\_event.c:8191)
08/01/2017 16:24:58.351, stderr:   File "C:\Python27\lib\site-packages\kivy\uix\widget.py", line 457, in on_touch_down
08/01/2017 16:24:58.355, stderr:     if child.dispatch('on_touch_down', touch):
08/01/2017 16:24:58.357, stderr:   File "kivy\_event.pyx", line 718, in kivy._event.EventDispatcher.dispatch (kivy\_event.c:8191)
08/01/2017 16:24:58.358, stderr:   File "C:\Python27\lib\site-packages\kivy\uix\widget.py", line 457, in on_touch_down
08/01/2017 16:24:58.361, stderr:     if child.dispatch('on_touch_down', touch):
08/01/2017 16:24:58.362, stderr:   File "kivy\_event.pyx", line 718, in kivy._event.EventDispatcher.dispatch (kivy\_event.c:8191)
08/01/2017 16:24:58.365, stderr:   File "C:\Python27\lib\site-packages\kivy\uix\widget.py", line 457, in on_touch_down
08/01/2017 16:24:58.367, stderr:     if child.dispatch('on_touch_down', touch):
08/01/2017 16:24:58.368, stderr:   File "kivy\_event.pyx", line 718, in kivy._event.EventDispatcher.dispatch (kivy\_event.c:8191)
08/01/2017 16:24:58.369, stderr:   File "C:\Python27\lib\site-packages\kivy\uix\widget.py", line 457, in on_touch_down
08/01/2017 16:24:58.372, stderr:     if child.dispatch('on_touch_down', touch):
08/01/2017 16:24:58.374, stderr:   File "kivy\_event.pyx", line 718, in kivy._event.EventDispatcher.dispatch (kivy\_event.c:8191)
08/01/2017 16:24:58.377, stderr:   File "C:\Python27\lib\site-packages\kivy\uix\behaviors\button.py", line 151, in on_touch_down
08/01/2017 16:24:58.378, stderr:     self.dispatch('on_press')
08/01/2017 16:24:58.381, stderr:   File "kivy\_event.pyx", line 714, in kivy._event.EventDispatcher.dispatch (kivy\_event.c:8146)
08/01/2017 16:24:58.382, stderr:   File "kivy\_event.pyx", line 1225, in kivy._event.EventObservers.dispatch (kivy\_event.c:14035)
08/01/2017 16:24:58.385, stderr:   File "kivy\_event.pyx", line 1109, in kivy._event.EventObservers._dispatch (kivy\_event.c:12816)
08/01/2017 16:24:58.388, stderr:   File "C:\Python27\lib\site-packages\kivy\lang\builder.py", line 64, in custom_callback
08/01/2017 16:24:58.390, stderr:     exec(__kvlang__.co_value, idmap)
08/01/2017 16:24:58.391, stderr:   File "C:\Users\MOLNAR_PH330_D1\Desktop\NZERO\python_code\nzero_testing_app.kv", line 187, in <module>
08/01/2017 16:24:58.394, stderr:     on_press: root.draw_graph("a1", self.text)
08/01/2017 16:24:58.395, stderr:   File "C:\Users\MOLNAR_PH330_D1\Desktop\NZERO\python_code\pstest.py", line 249, in draw_graph
08/01/2017 16:24:58.398, stderr:     bias_v.append(float(arr[i_bias_v]))
08/01/2017 16:24:58.401, stderr: ValueError: invalid literal for float(): 3.685174e-11
08/01/2017 16:24:58.403, stderr: 1.5
